torch/cuda/amp/grad_scaler.py

```diff
@@ -1,6 +1,7 @@
 from typing_extensions import deprecated
 
 import torch
+from torch.amp.grad_scaler import OptState
 
 __all__ = ["GradScaler"]
```

Comment:
Do we need to add back to `__all__` also?

Reply:
No because it doesn't come from here. So it is not a proper public API (And why I removed it in the first place).